index.d.ts

```diff
@@ -79,6 +79,11 @@ declare interface Props {
    * HTML attribute class
    */
   className?: string
+
+  /**
+   * Explicit children attribute required by React 18
```

Comment:
Since this is the public description of the prop I think it should document what to pass here, something like:

```suggestion
   * The children passed in is what will be rendered as the actual Tinder-style card
```